riotfile.py

```diff
@@ -909,6 +909,14 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="flask_login",
+            command="pytest {cmdargs} tests/contrib/flask_login",
+            pkgs={
+                "flask": "~=0.12.0",
```

Comment:
Is this right? The [flask-login setup.py](https://github.com/maxcountryman/flask-login/blob/0.6.2/setup.py) requires `Flask>=1.0.4`.

Reply:
Good catch. Fixed, thanks!